make.go

```diff
@@ -236,12 +236,16 @@ func createGitRepository(debsrc, gopkg, orig string) (string, error) {
 		return dir, err
 	}
 
-	if err := runGitCommandIn(dir, "config", "user.name", getDebianName()); err != nil {
-		return dir, err
+	if debianName := getDebianName(); debianName == "TODO" {
```

Comment:
I should add that the != to == change also negates the fix for my own project:
`dh-make-golang -allow_unknown_hoster github.com/google/cups-connector`